onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -95,7 +95,9 @@ def test_with_types_disabled_due_to_binary_size_concerns_filters():
     filters = ['^test_bitshift_right_uint16_cpu',
                '^test_bitshift_right_uint8_cpu',
                '^test_bitshift_left_uint16_cpu',
-               '^test_bitshift_left_uint8_cpu']
+               '^test_bitshift_left_uint8_cpu',
+               '^test_edge_pad_cpu.*',
+               '^test_reflect_pad_cpu.*']
 
```

Comment:
Don't think you need to wildcard the end of these test names. Probably doesn't matter too much as this is temporary given [https://github.com/microsoft/onnxruntime/pull/2179](https://github.com/microsoft/onnxruntime/pull/2179) is tagged with the 1.0 milestone.

Reply:
yes you are right. will let #2179 remove it altogether from exclusion.